appcenter-analytics/scripts/postlink.js

```diff
@@ -39,26 +23,8 @@ if (rnpmlink.ios.checkIfAppDelegateExists()) {
             console.log('Configuring AppCenter Analytics for iOS');
             return rnpmlink.ios.initAppCenterConfig();
         })
-        .then(() =>
-            inquirer.prompt([{
-                type: 'list',
-                name: 'whenToEnableAnalytics',
-                message: 'For the iOS app, should user tracking be enabled automatically?',
-                choices: [
-                    {
-                        name: 'Enable Automatically',
-                        value: 'ALWAYS_SEND'
-                    },
-                    {
-                        name: 'Enable in JavaScript',
-                        value: 'ENABLE_IN_JS'
-                    }]
-            }])
-        )
-        .then((iosAnswer) => {
-            const code = iosAnswer.whenToEnableAnalytics === 'ALWAYS_SEND' ?
-                '  [AppCenterReactNativeAnalytics registerWithInitiallyEnabled:true];  // Initialize AppCenter analytics' :
-                '  [AppCenterReactNativeAnalytics registerWithInitiallyEnabled:false];  // Initialize AppCenter analytics';
+        .then(() => {
+            const code = '  [AppCenterReactNativeAnalytics registerWithInitiallyEnabled:true];  // Initialize AppCenter analytics';
```

Comment:
This will overwrite the setting that will have to be manually changed by customer.
This must be added only if no line is there.